docs/rules/comma-style.md

```diff
@@ -1,4 +1,4 @@
-# Comma style (comma-style)
+# Enforce One True Comma Style (comma-style)
```

Comment:
FWIW I'm :-1: on this change. I'd be okay with "Enforce one comma style" or "Enforce consistent comma style". However, I don't want people to subconsciously link this to 1tbs ("one true brace style"), which refers to a particular style guide. (Similarly I would like the brace-style documents to avoid mentioning "one true" in their headers/high-level descriptions.)